.gitignore

```diff
@@ -40,6 +40,4 @@ venv/
 ENV/
 env.bak/
 venv.bak/
-ivy_dev/
-*venv/
-venv*
\ No newline at end of file
+ivy_dev/
```

Comment:
Can you please undo the changes here in this file :)

Reply:
Sure, I have reverted the changes in the gitignore file as you requested. The unnecessary line has been removed. Let me know if there's anything else I can assist with! :)